freeipa/src/main/resources/schema/app/20201203102940_CB-10140_add_accountId_to_freeipa_SaltSecurityConfig.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-10140 add accountId to freeipa SaltSecurityConfig
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE saltsecurityconfig ADD COLUMN IF NOT EXISTS accountid VARCHAR(255) DEFAULT 'undefined';
```

Comment:
Default value shouldn't be added like this. It will block the whole table. 
See https://dba.stackexchange.com/questions/60965/does-postgresql-optimize-adding-columns-with-non-null-defaults/216153#216153
Example: https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/schema/app/20201130155647_CB-9580_new_field_on_cluster:_embedded_db_is_configured_on_attached_disk_or_not.sql

Also I think we should set it to null.

Reply:
done